cmd/rigs/cmd/publish_data.go

```diff
@@ -202,7 +202,7 @@ func lookupsJob(s local.Store, jobID *int) wpool.Job {
 				ImageMediumAlphaName: "image_medium_alpha.png",
 				ImageThumbName:       "image_thumb.png",
 				ImageThumbAlphaName:  "image_thumb_alpha.png",
-				AnimationBaseURL:     "https://tableland.xyz/rigs/animate?id=", // TODO: Update when we know it.
+				AnimationBaseURL:     "https://rigs.tableland.xyz/",
```

Comment:
Note the animation base url used here. This value will be referenced in the metadata query templates below.